superheroes.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+ <html>
+   <head>
+     <meta charset="utf-8">
+     <!-- UI Sheets -->
+     <link rel="stylesheet" type="text/css" href="css/materialize.css">
+     <link rel="stylesheet" type="text/css" href="css/fonts.css">
+     <script type="text/javascript" src="js/sweetalert2.min.js"></script>
+     <!-- Include Font Awesome Stylesheet -->
+     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+     <!-- @0x48piraj custom social section stylesheet -->
+     <link rel="stylesheet" type="text/css" href="css/footer.css">
+     <!-- Engine Scripts -->
+     <script type="text/javascript" src="js/jquery.js"></script>
+     <script type="text/javascript" src="js/materialize.js"></script>
+     <script type="text/javascript" src="js/tilesSuperhero.js"></script>
+     <script type="text/javascript" src="js/levelsSuperhero.js"></script>
+     <script type="text/javascript" src="js/player.js"></script>
+     <script type="text/javascript" src="js/level.js"></script>
+     <link rel="stylesheet" href="css/tiles.css">
+     <title>Labyrinth</title>
+   </head>
+ <body>
+ <div class="navbar-fixed">
+ 	<nav>
+ 	  <div class="nav-wrapper">
+ 	    <a href="https://github.com/fossasia/labyrinth" class="brand-logo center">Labyrinth</a>
+ 	    <ul class="right hide-on-med-and-down">
+ 	      <!-- Sample link -->
+         <li><a href="index.html">Back to Lv.1</a></li>
+ 	      <li><a href="howtoplay.html">How to Play</a></li>
+ 	    </ul>
+ 	  </div>
+ 	</nav>
+ </div>
+ 
+ <!-- Data Card -->
+ <div class="row">
+     <div class="col m4 s8">
+       <div class="card blue-grey darken-1">
+         <div class="card-content white-text">
+           <span class="card-title">Statistics</span>
+           <p>Player Moved : <span id="playerMoves">0</span> Times</p>
+           <p></i>Lv.3 Forest by Arjun and Samyak</i></p>
+         </div>
+         <div class="card-action">
+           <a href="http://github.com/yashkumarverma/">UI by Yash Kumar Verma</a>
```

Comment:
Please do not copy the index file #161 This needs to be reverted and it is almost impossible to revert this act of copying.